AutonomousDemo/Road.cs

```diff
@@ -10,9 +10,15 @@ public class Road
     {
         public double Distance { get; set; }
         public double LaneWidth { get; set; }
+        public int SpeedLimit { get; set; }
```

Comment:
Probably deserves to be a double. (You might run into non-integer numbers if you map out a road system that crosses national boundaries, for example, where there's not a clean integer conversion between mph and km/h.)